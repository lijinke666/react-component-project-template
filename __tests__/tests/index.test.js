/*eslint-disable */
import React from "react";
import assert from "power-assert";
import { shallow } from "enzyme";

import YourComponent from "../../src";

describe("<YourComponent/>", () => {
  it("should render <YourComponent/> components", () => {
    const wrapper = shallow(<YourComponent />);
    expect(wrapper).toMatchSnapshot();
  });
  it("should find <h2> for <YourComponent/>", () => {
    const wrapper = shallow(<YourComponent />);
    assert(wrapper.find("h2").length === 1);
  });
  it("should find <p> for <YourComponent/>", () => {
    const wrapper = shallow(<YourComponent />);
    assert(wrapper.find("p").length === 1);
  });
});
