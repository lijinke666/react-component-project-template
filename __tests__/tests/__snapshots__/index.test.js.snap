// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<YourComponent/> should render <YourComponent/> components 1`] = `
<React.Fragment>
  <h2
    className="title"
    key="title"
  >
    react-component-project-template
  </h2>
  <p
    key="p"
  >
    By:
    <a
      href="https://github.com/lijinke666/dawdler"
      target="_blank"
    >
      Dawdler
    </a>
  </p>
</React.Fragment>
`;
